submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,272 @@
+| ![Assembling machine 2.png](https://wiki.factorio.com/images/thumb/Assembling_machine_2.png/32px-Assembling_machine_2.png) Assembling machine 2 | 0.150*                         | ![Rocket fuel.png](https://wiki.factorio.com/images/thumb/Rocket_fuel.png/32px-Rocket_fuel.png)![Uranium fuel cell.png](https://wiki.factorio.com/images/thumb/Uranium_fuel_cell.png/32px-Uranium_fuel_cell.png)                                                                                                                                                                                                                                                                                                                                             | 0.75*          |
+| ![Centrifuge.png](https://wiki.factorio.com/images/thumb/Centrifuge.png/32px-Centrifuge.png) Centrifuge                                         | 0.350                          | ![Nuclear fuel.png](https://wiki.factorio.com/images/thumb/Nuclear_fuel.png/32px-Nuclear_fuel.png)                                                                                                                                                                                                                                                                                                                                                                                                                                                           | 1              |
+
+*If we use assembling machine 3 instead, the power use would be 0.375MW and crafting speed 1.25.
+
+The total processing energy ("E") needed for a conversion can be calculated using the machine power use ("P") multiplied by the total crafting time for all of the input ("T"). This total crafting time is the duration of one crafting cycle ("d") divided by the machine crafting speed ("s"), multiplied by the number of crafting cycles ("c") to cover the entire input quantity. Hence we summarize processing energy consumption as:
+
+`E = P * T = P * d * c / s`
+
+While we calculate these values without modules to see the "default" situation, I would highly recommend to fill these machines with at least efficiency 1 modules in practice, as it would reduce E by 60%-80%. Other module combinations can offer even better results, like extra yield from speed-productivity synergy. [This guide]([Tutorial:Producing power from oil - Factorio Wiki](https://wiki.factorio.com/Tutorial:Producing_power_from_oil)) on the official wiki looks into these in detail.
```

Comment:
is it at this moment that you should discuss about adding modules? because modules could/should be placed into all machines, being efficience/productivity or speed, having impact on the whole chain and maybe reversing some conclusions? then, if this is true, I would remove all module from the initial calculation, and discuss them into a dedicated section, with maybe "the best module setup to maximise <watéver> from the chain"

Reply:
I agree that talking about modules right after emphasizing that they are not used is unnecessary. I'll leave this discussion to only the end. 

I had not thought that they might be effective enough to reverse the conclusions, but now that you mention it, I realized that one can indeed increase rocket fuel yield by 40% with productivity modules and that would probably reverse that conclusion. I think I'll look into it a bit more and rewrite some conclusions...